controllers/publish-pane.js

```diff
@@ -14,14 +15,24 @@ module.exports = function () {
     },
 
     onPublishNow: function () {
+      pane.close();
+      progress.start('green');
+
       return validation.validate(rules).then(function (errors) {
         if (errors.length) {
-          pane.close();
+          progress.done('red');
```

Comment:
Code red!  Hey, progress bar, be red now.

Really, shouldn't the progress bar be responsible for how it looks?  Making the publish pane responsible for how _someone else_ looks is an anti-pattern.